1-the-basics/index.html

```diff
@@ -18,9 +18,11 @@ <h2>Hobbies</h2>
             <li>Writing code !</li>
             <li>Watch movies</li>
             <li>Reading</li>
+            <li>Learning new things</li>
+            <li>Watch YouTube videos</li>
         </ul>
 
-        <p>Hi I'm Oscar :)</p>
+        <p>Hi I'm Oscar :) <br> How you doing</p>
         <img src="img/photo.jpg" alt="King" width="50%" height="230px">
```

Comment:
Once we start with CSS you will be able to notice that it's better to manage the image dimensions outside of the html attributes.